staging/src/k8s.io/apiextensions-apiserver/test/integration/fixtures/resources.go

```diff
@@ -275,6 +276,63 @@ func CreateNewCustomResourceDefinition(crd *apiextensionsv1beta1.CustomResourceD
 	return crd, nil
 }
 
+// CreateNewV1CustomResourceDefinitionWatchUnsafe creates the CRD and makes sure
+// the apiextension apiserver has installed the CRD. But it's not safe to watch
+// the created CR. Please call CreateNewV1CustomResourceDefinition if you need to
+// watch the CR.
+func CreateNewV1CustomResourceDefinitionWatchUnsafe(v1CRD *apiextensionsv1.CustomResourceDefinition, apiExtensionsClient clientset.Interface) (*apiextensionsv1.CustomResourceDefinition, error) {
```

Comment:
Strangely, the addition of Unsafe makes me feel less safe.

Reply:
Aren't these supposed to be in a separate bug fix?

Reply:
Are you pre-fixing this for the general issue?  Or it only applies here?

Reply:
this is just a copy of the existing v1beta1 method to let you create a v1 version. it is fine if you have a test that doesn't use watch (this just skips waiting for watch caches to fill)

Reply:
it's not really a bug, if you list/watch while the watch cache is establishing, you can get a 410 from the first watch request... a normal client will just re-list/re-watch, but deterministic watch tests hard fail.